website/src/blogs/hacktoberfest-2020-contribute-to-openebs.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Hacktoberfest 2020 - Contribute to OpenEBS"
+author: MayaData Marketing
+author_info: Mayadata Marketing Team
+date: 30-09-2020
+tags: OpenEBS
+excerpt: Hacktoberfest 2020 is almost here. Contribute to open source project, OpenEBS, to win exciting swag.
+---
+
+### Hacktoberfest returns! Contribute to OpenEBS and win exciting swag
+
+The seventh annual [***Hacktoberfest ***](https://hacktoberfest.digitalocean.com/)celebration is almost here, and we at *OpenEBS *are happy to be participating in the contest once again. In August 2017, the OpenEBS community began growing and building a strong foundation for an open source project.
```

Comment:
```suggestion
The seventh annual [***Hacktoberfest ***](https://hacktoberfest.digitalocean.com/) celebration is almost here, and we at *OpenEBS *are happy to be participating in the contest once again. In August 2017, the OpenEBS community began growing and building a strong foundation for an open source project.
```

Reply:
it seems that `**` is not converting to *italic* in blog post

<img width="912" alt="Screenshot 2021-07-01 at 10 48 06 AM" src="https://user-images.githubusercontent.com/6524419/124069693-366da800-da5a-11eb-84c4-e5eadddd0788.png">

Reply:
Done.